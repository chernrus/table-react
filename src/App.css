span {
  display: block;
}

button {
  background-color: #DCDCDC;
  border: none;
  height: 40px;
  min-width: 100px;
  margin-left: 10px;
}

button:hover {
  background-color: #F0F0F0;
}

.small-data-btn {
  margin-left: 0;
}

.Error-message {
  font-size: 18px;
  font-weight: bold;
  color: red;
}

.Load-data-panel,
.Error-message,
.Table-container
 {
  margin: 10px;
}

.Filter-panel {
  margin-bottom: 10px;
}

.Table-container {
  width: 80%;
}

.Spinner__circle {
  border: 16px solid #f3f3f3;
  border-radius: 50%;
  border-top: 16px solid #A9A9A9;
  width: 120px;
  height: 120px;
  -webkit-animation: spin 2s linear infinite; /* Safari */
  animation: spin 2s linear infinite;
}

.Spinner__layout {
  position: absolute;
  z-index: 100;

  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  background-color: #fff;
  width: 100%;
  height: 100%;
  text-align: center;
}

/* Safari */
@-webkit-keyframes spin {
  0% { -webkit-transform: rotate(0deg); }
  100% { -webkit-transform: rotate(360deg); }
}

@keyframes spin {
  0% { transform: rotate(0deg); }
  100% { transform: rotate(360deg); }
}

table {
  border-collapse: collapse;
  width: 100%;
}

table, th, td {
  border: 1px solid black;
}

td {
  padding: 5px;
}

th:hover {
  background-color: #A9A9A9;
}

.Table-container__row {
  cursor: pointer;
}

.Table-container__row:hover {
  background-color: #DCDCDC;
}

.sortDirection1::after {
  content: '\2BC5';
}

.sortDirection-1::after {
  content: '\2BC6';
}

.pagination {
  display: inline-block;
  padding: 0;
}

.pagination li {
  color: black;
  float: left;
  padding: 8px 16px;
  list-style-type: none;
}

.pagination li.active {
  background-color: #A9A9A9;
  color: white;
}

.pagination li:hover:not(.active) {
  background-color: #ddd;
}

.User-profile {
  border: 1px solid;
  padding: 10px;
}
