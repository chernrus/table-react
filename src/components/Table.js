import React, {Component} from 'react';

const Table = (props) => {
  // data = [data[0]];
  console.log(props);


    return (
      <div className="Table-containe">
        <table>
          <thead>
            <tr>
              <th >id</th>
              <th >firstName</th>
              <th >lastName</th>
              <th >email</th>
              <th >phone</th>
            </tr>
          </thead>
          <tbody>
          {props.data.map(userData => (
            <tr onClick={() => {this.showUserProfile(userData)}}>
              <td>{userData.id}</td>
              <td>{userData.firstName}</td>
              <td>{userData.lastName}</td>
              <td>{userData.email}</td>
              <td>{userData.phone}</td>
            </tr>
          ))}
          </tbody>
        </table>
      </div>
    );

}

export default Table;
