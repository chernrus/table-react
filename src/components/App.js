import React, { Component } from 'react';
import '../App.css';
import Spinner from './Spinner';
import Table from './Table';

class App extends Component {
  state = {
    data: [],
    error: null,
    isLoading: false,
    userProfile: null
  }

  loadData = (type) => {

    this.setState({ isLoading: true})

    const smallDataUrl = 'http://www.filltext.com/?rows=32&id={number|1000}&firstName={firstName}&lastName={lastName}&email={email}&phone={phone|(xxx)xxx-xx-xx}&address={addressObject}&description={lorem|32}',
      bigDataUrl = 'http://www.filltext.com/?rows=1000&id={number|1000}&firstName={firstName}&delay=3&lastName={lastName}&email={email}&phone={phone|(xxx)xxx-xx-xx}&address={addressObject}&description={lorem|32}';

    let url = '';

    if(type === 'small-data-value') {
      url = smallDataUrl
    }
    if(type === 'big-data-value') {
      url = bigDataUrl
    }

    fetch(url)
      .then(response => {
          return response.json();
      })
      .then(data => this.setState({data: data}))
      .catch(error => this.setState({ error: error}))
      .then(() => this.setState({ isLoading: false}));
  }

  sortData = (column) => {
    console.log(column);
  }

  showUserProfile = (userData) => {
    this.setState({userProfile: userData})
  }

  render() {
    const {
      isLoading,
      data
    } = this.state

    // let filteredData = data;

    console.log(data);
    return (

      <div className="App">
        {isLoading && <Spinner />}

        <div className="Setting-panel">
          <div className="Setting-panel__mode-coice">
            <p>Выбор объема данных</p>
            <button onClick={() => {this.loadData('small-data-value')}}>Малый объем данных</button>
            <button onClick={() => {this.loadData('big-data-value')}}>Большой объем данных</button>
          </div>
          <div className="Setting-panel__filter-panel">
            <input type="text" placeholder="Введите подстроку"/>
            <button>Найти</button>
          </div>
        </div>
        
        <div className="Table-container">
          <table>
            <thead>
              <tr>
                <th onClick={() => {this.sortData('id')}}>id</th>
                <th onClick={() => {this.sortData('firstName')}}>firstName</th>
                <th onClick={() => {this.sortData('lastName')}}>lastName</th>
                <th onClick={() => {this.sortData('email')}}>email</th>
                <th onClick={() => {this.sortData('phone')}}>phone</th>
              </tr>
            </thead>
            <tbody>
            {data.map(userData => (
              <tr onClick={() => {this.showUserProfile(userData)}}>
                <td>{userData.id}</td>
                <td>{userData.firstName}</td>
                <td>{userData.lastName}</td>
                <td>{userData.email}</td>
                <td>{userData.phone}</td>
              </tr>
            ))}
            </tbody>
          </table>
        </div>
      </div>
    );
  }
}

export default App;
