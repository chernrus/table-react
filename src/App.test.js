import React from 'react';
import ReactDOM from 'react-dom';
import App from './components/App';
import Table from './components/Table';
import renderer from 'react-test-renderer';

const smallDataJSON = '[{"id":454,"firstName":"Brett","lastName":"Taylor","email":"BJalowiecki@nullam.net","phone":"(685)196-1577","address":{"streetAddress":"7508 Sit Ln","city":"Saucier","state":"MS","zip":"62337"},"description":"pulvinar massa sagittis tortor sit at tellus vitae elementum nunc aenean nec pulvinar placerat libero sit lacus lectus massa sollicitudin sit sollicitudin amet lacus sed facilisis vestibulum odio aliquam vel non sit"},{"id":221,"firstName":"Verna","lastName":"Schlottman","email":"MFaurest@tempor.gov","phone":"(371)456-8139","address":{"streetAddress":"8457 Lorem Ct","city":"Vienna","state":"LA","zip":"11632"},"description":"sollicitudin augue adipiscing tortor id elit consectetur etiam tincidunt quis tellus massa porttitor id lorem lacus placerat hendrerit hendrerit massa sapien et sagittis mi lacus fringilla amet tincidunt vestibulum consequat elit sed"},{"id":827,"firstName":"Brien","lastName":"Blayne","email":"JBorisyuk@non.gov","phone":"(972)837-6441","address":{"streetAddress":"4035 Lorem Dr","city":"Bremen","state":"OK","zip":"32340"},"description":"elementum porttitor ante et nullam massa vestibulum aliquam vel facilisis id turpis odio sollicitudin tincidunt vitae pretium quis nullam adipiscing sed scelerisque eros curabitur consequat magna ante sollicitudin nec eget aenean nunc"},{"id":161,"firstName":"Jerline","lastName":"Noe","email":"VWestrate@donec.ly","phone":"(157)213-9201","address":{"streetAddress":"2488 Morbi Ln","city":"Lees Summit","state":"NM","zip":"93855"},"description":"magna ante amet hendrerit amet vitae placerat donec et pulvinar consequat sed eget amet curabitur nec sed velit sit odio sit vel consectetur sollicitudin sed tincidunt magna curabitur id dolor placerat ac"},{"id":323,"firstName":"Dwayne","lastName":"Ibarrondo","email":"BYork@eros.com","phone":"(102)225-1788","address":{"streetAddress":"7316 Odio Dr","city":"Kansas City","state":"NE","zip":"63782"},"description":"tortor curabitur mattis dolor mi vitae amet at vestibulum massa at malesuada convallis turpis aliquam sagittis molestie ante at convallis libero at ipsum nunc ac pretium sollicitudin massa libero elit adipiscing pretium"},{"id":89,"firstName":"Wazir","lastName":"Mazza","email":"LBeebe@elementum.gov","phone":"(253)013-7725","address":{"streetAddress":"1999 Fringilla Rd","city":"Danbury","state":"WY","zip":"45930"},"description":"ante vel sit vitae tempor nullam fringilla nec tempor amet aliquam tincidunt nullam sit magna placerat egestas pharetra libero porttitor eros molestie amet malesuada mattis dolor libero suspendisse ac malesuada turpis scelerisque"},{"id":92,"firstName":"Maxime","lastName":"Cook","email":"PKarneol@ipsum.com","phone":"(145)350-4932","address":{"streetAddress":"4217 Pharetra Ave","city":"Cambridge","state":"WY","zip":"69859"},"description":"amet nunc sit ipsum lorem sit dui placerat et et lacus et magna sagittis placerat tortor nec dolor at orci rutrum eget mattis tortor donec amet elit odio odio eros vitae orci"},{"id":609,"firstName":"Corretta","lastName":"Vanderlinden","email":"NDerkson@pulvinar.io","phone":"(548)539-0437","address":{"streetAddress":"388 Placerat Ln","city":"Killeen","state":"PA","zip":"34992"},"description":"mattis at etiam mi etiam orci donec molestie lacus sit vitae nec ac placerat convallis in hendrerit molestie sagittis sit pulvinar sit nec at mattis tortor vitae egestas suspendisse fringilla nunc convallis"},{"id":184,"firstName":"Javier","lastName":"Pointelin","email":"ASilverstein@augue.org","phone":"(793)662-3943","address":{"streetAddress":"1688 Fringilla Dr","city":"Baton Rouge","state":"FL","zip":"40540"},"description":"sed id tempor sit sollicitudin id odio pretium tortor lacus mattis dolor aliquam nec sed ipsum massa at pulvinar id dolor risus lorem dui libero magna dolor turpis sed nec nec scelerisque"},{"id":337,"firstName":"Alina","lastName":"Tinker","email":"JRamsey@tellus.net","phone":"(288)488-1915","address":{"streetAddress":"720 Vel Ave","city":"Oakland","state":"WI","zip":"83808"},"description":"augue sit vestibulum fringilla consequat massa porttitor lacus magna ipsum vel amet elit vitae egestas amet tellus dui amet sit orci at in sit amet hendrerit orci libero augue sed rutrum vestibulum"},{"id":673,"firstName":"Aely","lastName":"Hutchinson","email":"VDixon@tortor.ly","phone":"(651)291-5023","address":{"streetAddress":"882 Consectetur Rd","city":"Senoia","state":"MA","zip":"19587"},"description":"nec sapien amet malesuada tincidunt nec pulvinar tortor mattis tortor in amet mattis rutrum suspendisse pulvinar tincidunt et ac lacus tortor vestibulum porttitor pharetra porta hendrerit dui sed vitae lorem quis elementum"},{"id":723,"firstName":"Gerarld","lastName":"Hughes","email":"KFaurest@facilisis.io","phone":"(668)030-6390","address":{"streetAddress":"7807 Dui Ave","city":"Chambersburg","state":"AL","zip":"27667"},"description":"hendrerit curabitur vestibulum ante quis sollicitudin adipiscing vel mi et at magna et sollicitudin molestie hendrerit sed eget nec tellus velit magna egestas lacus ipsum nunc tortor curabitur id nec adipiscing elit"},{"id":191,"firstName":"Stacie","lastName":"Bowers","email":"BSholtz@libero.org","phone":"(843)764-7362","address":{"streetAddress":"7185 Amet St","city":"Tonasket","state":"NC","zip":"77103"},"description":"amet et sagittis odio turpis tortor turpis vel adipiscing consequat sollicitudin eros placerat suspendisse dolor scelerisque sapien augue dolor sollicitudin amet nullam nunc dolor et aliquam in ante vitae convallis tellus nec"},{"id":159,"firstName":"Gus","lastName":"Krout","email":"SKinnear@magna.net","phone":"(963)928-2109","address":{"streetAddress":"4997 Aenean Rd","city":"Janesville","state":"VT","zip":"16721"},"description":"facilisis dolor elementum aenean fringilla lacus tempor augue at elementum pharetra sit pulvinar massa et porta id mattis rutrum rutrum adipiscing pulvinar ipsum et porttitor quis orci porttitor elit tincidunt donec etiam"},{"id":701,"firstName":"Maegan","lastName":"Chambers","email":"SMuth@vestibulum.org","phone":"(390)873-9469","address":{"streetAddress":"5769 Ipsum Ln","city":"Boulder","state":"TX","zip":"54504"},"description":"tincidunt mattis scelerisque orci augue malesuada elementum porta magna lectus hendrerit malesuada dolor sit at rutrum amet odio sit tincidunt sapien porttitor pulvinar egestas vestibulum orci sit magna orci ipsum pulvinar egestas"},{"id":51,"firstName":"LaShonda","lastName":"Shrestha","email":"ABurkes@eget.ly","phone":"(561)953-3584","address":{"streetAddress":"9060 Turpis Rd","city":"Westwood","state":"LA","zip":"58918"},"description":"curabitur curabitur aliquam fringilla etiam donec porta porta massa turpis amet lacus lacus dolor ipsum tortor vestibulum lacus magna sit hendrerit lectus aliquam curabitur amet dui tellus adipiscing vitae tincidunt massa tincidunt"},{"id":757,"firstName":"Tiziana","lastName":"Larrabee","email":"NHampton@amet.ly","phone":"(283)180-8710","address":{"streetAddress":"3010 Tempor Ave","city":"Carrollton","state":"TN","zip":"79854"},"description":"dolor lectus porta eros dui egestas in sed facilisis aenean placerat mi tortor eget non consequat dolor magna in quis sollicitudin dolor sollicitudin lorem molestie sed dui vestibulum velit dolor amet magna"},{"id":327,"firstName":"Eden","lastName":"Ragusa","email":"APlanty@vestibulum.io","phone":"(852)979-0937","address":{"streetAddress":"1084 Placerat Ave","city":"Warsaw","state":"OR","zip":"94213"},"description":"adipiscing mattis porttitor curabitur tincidunt placerat morbi consequat curabitur magna fringilla lacus risus donec nec pretium amet curabitur augue dolor orci vestibulum dolor aliquam malesuada nullam rutrum pulvinar ac id in dolor"},{"id":108,"firstName":"Usha","lastName":"Rock","email":"YArnold@sollicitudin.gov","phone":"(930)774-2526","address":{"streetAddress":"166 Tellus Dr","city":"Panama City Beach","state":"IA","zip":"35300"},"description":"facilisis vestibulum vel tortor facilisis mattis fringilla at eget ante ipsum mattis ipsum odio lorem pretium sapien vitae aliquam consequat tempor vestibulum non ante nec pharetra sagittis molestie ipsum sed sapien massa"},{"id":536,"firstName":"Heriberto","lastName":"Mize","email":"JPilgrim@sed.ly","phone":"(886)529-4554","address":{"streetAddress":"3139 Egestas Ln","city":"Whitehouse","state":"MS","zip":"99227"},"description":"amet neque tellus donec quis sit convallis libero sit lectus sed dolor nunc ipsum non orci porttitor malesuada pharetra velit velit eros aliquam nec consequat amet augue sit non sollicitudin vestibulum consectetur"},{"id":576,"firstName":"Ayeola","lastName":"Rossi","email":"AMohamed@elit.net","phone":"(173)640-4996","address":{"streetAddress":"2509 Pulvinar Ln","city":"Wichita","state":"OR","zip":"80880"},"description":"nec elit id hendrerit placerat tincidunt at vitae dolor facilisis augue magna tellus ipsum donec vel vitae sit dolor malesuada magna placerat vestibulum fringilla tincidunt dui sed tincidunt placerat nec risus risus"},{"id":560,"firstName":"Brooks","lastName":"Llaneza","email":"VTreptow@malesuada.net","phone":"(241)349-5611","address":{"streetAddress":"43 Tortor Dr","city":"Rexburg","state":"KS","zip":"92012"},"description":"ac dolor massa sollicitudin aliquam adipiscing pharetra pulvinar tortor rutrum sit tincidunt sed vestibulum molestie sollicitudin lacus nullam pretium dolor curabitur lacus orci mattis ac lacus sollicitudin pulvinar sit donec dui amet"},{"id":790,"firstName":"Jerry","lastName":"Gutsteinberg","email":"KGiera@pretium.ly","phone":"(606)351-9483","address":{"streetAddress":"2697 Sed Ct","city":"Saltilo","state":"ID","zip":"35044"},"description":"scelerisque sed mi donec vel lectus lacus magna curabitur rutrum pulvinar eros etiam id nec lacus sit at massa pulvinar mattis vitae tincidunt pharetra et eros pretium ipsum eget sed tortor ipsum"},{"id":321,"firstName":"Jody","lastName":"Anastasiades","email":"ARannila@aliquam.io","phone":"(597)371-1917","address":{"streetAddress":"9249 Odio Dr","city":"Old Bridge","state":"IA","zip":"37920"},"description":"lectus amet dolor et odio sagittis egestas nunc rutrum vitae elementum tortor sollicitudin eget mi mattis id elementum turpis elit porttitor et eget hendrerit nullam massa suspendisse porttitor ante hendrerit morbi odio"},{"id":664,"firstName":"Sheba","lastName":"Mortimore","email":"RVicari@etiam.net","phone":"(750)654-8985","address":{"streetAddress":"5998 Morbi Ct","city":"Santa Clarita","state":"NE","zip":"68518"},"description":"adipiscing convallis eros at et curabitur etiam sollicitudin vitae libero ac sed augue curabitur amet rutrum sit consequat vitae mattis elementum pharetra nunc consectetur ac scelerisque ac sed hendrerit tellus suspendisse etiam"},{"id":480,"firstName":"Kamaniee","lastName":"Connor","email":"RJeschke@morbi.ly","phone":"(525)745-0812","address":{"streetAddress":"2904 Mattis Dr","city":"Sebring","state":"NE","zip":"41274"},"description":"sed augue facilisis curabitur nec sapien velit magna nullam scelerisque augue ipsum scelerisque sit consectetur dolor sed ac hendrerit scelerisque at sed vitae sollicitudin malesuada turpis tortor velit fringilla massa id elit"},{"id":935,"firstName":"Toula","lastName":"Bario","email":"JPeltonen@aenean.org","phone":"(433)381-5542","address":{"streetAddress":"4414 Nec Rd","city":"Rexburg","state":"TX","zip":"77850"},"description":"augue vitae at rutrum adipiscing mattis suspendisse aliquam quis morbi suspendisse at sapien nunc dui mattis suspendisse augue dolor curabitur porta tempor egestas turpis at lacus nec facilisis ante tempor tincidunt lacus"},{"id":874,"firstName":"Axay","lastName":"Soo","email":"DAxe@eget.ly","phone":"(122)765-9147","address":{"streetAddress":"2081 Libero Ct","city":"Fargo","state":"DC","zip":"25955"},"description":"sit aenean tincidunt ac dolor et orci pulvinar mi tortor orci lacus nec tempor lectus aenean consectetur malesuada massa lacus pretium massa vitae ante neque sed sed magna odio elementum porta magna"},{"id":67,"firstName":"Jack","lastName":"Zamora","email":"DFeger@et.ly","phone":"(337)826-5454","address":{"streetAddress":"8838 Sit Ave","city":"Faribault","state":"DC","zip":"48449"},"description":"et lacus ipsum sollicitudin odio sollicitudin augue vitae vitae ante massa morbi rutrum at placerat sed lacus morbi neque nunc curabitur in aliquam ac mattis sed odio neque amet nunc etiam pulvinar"},{"id":714,"firstName":"Nadirah","lastName":"Lowe","email":"ALenze@magna.ly","phone":"(885)879-5545","address":{"streetAddress":"4161 Lacus Ave","city":"Ligonier","state":"MS","zip":"71505"},"description":"curabitur suspendisse risus lacus facilisis non sit sit ac augue tortor sit tortor nunc magna id sit amet dolor et amet sed nec et velit nec ipsum odio amet libero facilisis molestie"},{"id":207,"firstName":"Morticia","lastName":"Carpenter","email":"JPeltonen@nec.gov","phone":"(657)901-4287","address":{"streetAddress":"9495 Ac Ave","city":"St Augustine","state":"SC","zip":"40355"},"description":"magna sed vestibulum vestibulum nullam lorem lacus magna aliquam magna amet placerat sed nec odio lectus amet odio sagittis pulvinar adipiscing vestibulum magna nunc placerat tortor lacus augue velit sit lacus curabitur"},{"id":880,"firstName":"Wanda","lastName":"Hodgson","email":"RGrimm@quis.org","phone":"(362)687-0389","address":{"streetAddress":"849 Amet Dr","city":"Santa Monica","state":"UT","zip":"58462"},"description":"pharetra tortor magna amet lorem sit dolor placerat id magna tortor nec vitae amet fringilla adipiscing elementum ipsum nec id tellus ac at vel morbi rutrum dui hendrerit mi non curabitur non"}]';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('renders correctly', () => {
  const data = JSON.parse(smallDataJSON);
  const tree = renderer
    .create(<Table data={data}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();

});
