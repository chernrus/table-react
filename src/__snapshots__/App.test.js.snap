// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Table-container"
>
  <div
    className="Filter-panel"
  >
    <span>
      Фильтр
    </span>
    <input
      onChange={[Function]}
      placeholder="Введите подстроку"
      type="text"
      value=""
    />
    <button
      onClick={[Function]}
    >
      Найти
    </button>
  </div>
  <table>
    <thead>
      <tr
        className="Table-container__headRow"
      >
        <th
          className={null}
          onClick={[Function]}
        >
          id
        </th>
        <th
          className={null}
          onClick={[Function]}
        >
          firstName
        </th>
        <th
          className={null}
          onClick={[Function]}
        >
          lastName
        </th>
        <th
          className={null}
          onClick={[Function]}
        >
          email
        </th>
        <th
          className={null}
          onClick={[Function]}
        >
          phone
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          454
        </td>
        <td>
          Brett
        </td>
        <td>
          Taylor
        </td>
        <td>
          BJalowiecki@nullam.net
        </td>
        <td>
          (685)196-1577
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          221
        </td>
        <td>
          Verna
        </td>
        <td>
          Schlottman
        </td>
        <td>
          MFaurest@tempor.gov
        </td>
        <td>
          (371)456-8139
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          827
        </td>
        <td>
          Brien
        </td>
        <td>
          Blayne
        </td>
        <td>
          JBorisyuk@non.gov
        </td>
        <td>
          (972)837-6441
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          161
        </td>
        <td>
          Jerline
        </td>
        <td>
          Noe
        </td>
        <td>
          VWestrate@donec.ly
        </td>
        <td>
          (157)213-9201
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          323
        </td>
        <td>
          Dwayne
        </td>
        <td>
          Ibarrondo
        </td>
        <td>
          BYork@eros.com
        </td>
        <td>
          (102)225-1788
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          89
        </td>
        <td>
          Wazir
        </td>
        <td>
          Mazza
        </td>
        <td>
          LBeebe@elementum.gov
        </td>
        <td>
          (253)013-7725
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          92
        </td>
        <td>
          Maxime
        </td>
        <td>
          Cook
        </td>
        <td>
          PKarneol@ipsum.com
        </td>
        <td>
          (145)350-4932
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          609
        </td>
        <td>
          Corretta
        </td>
        <td>
          Vanderlinden
        </td>
        <td>
          NDerkson@pulvinar.io
        </td>
        <td>
          (548)539-0437
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          184
        </td>
        <td>
          Javier
        </td>
        <td>
          Pointelin
        </td>
        <td>
          ASilverstein@augue.org
        </td>
        <td>
          (793)662-3943
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          337
        </td>
        <td>
          Alina
        </td>
        <td>
          Tinker
        </td>
        <td>
          JRamsey@tellus.net
        </td>
        <td>
          (288)488-1915
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          673
        </td>
        <td>
          Aely
        </td>
        <td>
          Hutchinson
        </td>
        <td>
          VDixon@tortor.ly
        </td>
        <td>
          (651)291-5023
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          723
        </td>
        <td>
          Gerarld
        </td>
        <td>
          Hughes
        </td>
        <td>
          KFaurest@facilisis.io
        </td>
        <td>
          (668)030-6390
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          191
        </td>
        <td>
          Stacie
        </td>
        <td>
          Bowers
        </td>
        <td>
          BSholtz@libero.org
        </td>
        <td>
          (843)764-7362
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          159
        </td>
        <td>
          Gus
        </td>
        <td>
          Krout
        </td>
        <td>
          SKinnear@magna.net
        </td>
        <td>
          (963)928-2109
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          701
        </td>
        <td>
          Maegan
        </td>
        <td>
          Chambers
        </td>
        <td>
          SMuth@vestibulum.org
        </td>
        <td>
          (390)873-9469
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          51
        </td>
        <td>
          LaShonda
        </td>
        <td>
          Shrestha
        </td>
        <td>
          ABurkes@eget.ly
        </td>
        <td>
          (561)953-3584
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          757
        </td>
        <td>
          Tiziana
        </td>
        <td>
          Larrabee
        </td>
        <td>
          NHampton@amet.ly
        </td>
        <td>
          (283)180-8710
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          327
        </td>
        <td>
          Eden
        </td>
        <td>
          Ragusa
        </td>
        <td>
          APlanty@vestibulum.io
        </td>
        <td>
          (852)979-0937
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          108
        </td>
        <td>
          Usha
        </td>
        <td>
          Rock
        </td>
        <td>
          YArnold@sollicitudin.gov
        </td>
        <td>
          (930)774-2526
        </td>
      </tr>
      <tr
        className="Table-container__row"
        onClick={[Function]}
      >
        <td>
          536
        </td>
        <td>
          Heriberto
        </td>
        <td>
          Mize
        </td>
        <td>
          JPilgrim@sed.ly
        </td>
        <td>
          (886)529-4554
        </td>
      </tr>
    </tbody>
  </table>
  <ul
    className="pagination"
  >
    <li
      className="disabled"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        «
      </a>
    </li>
    <li
      className="disabled"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        ⟨
      </a>
    </li>
    <li
      className="active"
      onClick={[Function]}
    >
      <a
        className="undefined"
        href="#"
      >
        1
      </a>
    </li>
    <li
      className=""
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        2
      </a>
    </li>
    <li
      className=""
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        ⟩
      </a>
    </li>
    <li
      className=""
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        »
      </a>
    </li>
  </ul>
</div>
`;
